phase2.Rmd

```diff
@@ -148,17 +154,45 @@ str(combined)
 # so... honestly, i'm missing too many sentiment analysis vars for 
 # imputation to be of any use here. BUT we can try imputing ratings here!
 # what would be a good way to impute?
-summary(combined$Rating)
+# But we can confirm if it's MAR wrt to starting alphabet
+
+combined_missing_polarity <- combined %>% mutate(missing_median_polarity = is.na(median_polarity))
+
+combined_too_old_or_new <- combined_missing_polarity %>% filter(grepl("^[A-H]", App)) %>% pull(missing_median_polarity)
+
+combined_regular <- combined_missing_polarity %>% filter(grepl("^[I-Z]", App)) %>% pull(missing_median_polarity)
```

Comment:
Wrong variable names -- can you swap those? These should be ah and iz.

Reply:
Also I think `^[A-H]` and `^[^A-H]` suffices right? We can pass a flag to ignore case to `grepl` per doc to cover lowercase instances that might be in the data set, but if you do a not here, instead of grepping for H-Z, then you get the numeric and special characters too. https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/grep

Reply:
Should suffice I thought about it but didnot see any names with lowercase start